### ISSUE TITLE
Tutorial suggestion

### ISSUE THREAD

**Issue by marwenbk** [aguilaair/updat#19]
Thank you @aguilaair for all the effort you are making for the open source community,
as you have noticed, such package usage is still far from beginner friendly, 
making youtube tutorial about the required steps to build and ship ( even the example app ) will be life saving for many. what do you think?

**Comment by aguilaair** [comment]
Definitely! I currently do not have the time, but if someone has a chance to do it it would be super appreciated 🍡

**Comment by Filvanecek** [comment]
Hi thanks for you job. I have a one question. Is there any option to set prerelease version downloadable too?

**Comment by aguilaair** [comment]
@Filvanecek check https://stackoverflow.com/questions/67688662/how-can-i-get-the-latest-pre-release-release-for-my-github-repo-bash

**Comment by Filvanecek** [comment]
> @Filvanecek check https://stackoverflow.com/questions/67688662/how-can-i-get-the-latest-pre-release-release-for-my-github-repo-bash or https://docs.github.com/en/free-pro-team@latest/rest/releases/releases?apiVersion=2022-11-28#list-releases

sorry... but not too much usefull for me :D Im not using git . So i have on FTP json file with latest version. Im trying do 4th number available only for me. so if in json file is 1.1.1 == show update for everyone, if json file contain 1.1.1.1, show update only for me

**Comment by aguilaair** [comment]
@Filvanecek I would use the getLatestVersion and just check the way you explained it here

```dart
getLatestVersion: () async {
   // Github gives us a super useful latest endpoint, and we can use it to get the latest stable release
   final data = await http.get(Uri.parse(
      "https://api.github.com/repos/aguilaair/updat/releases/latest",
   ));

   // Return the tag name, which is always a semantically versioned string.
   return jsonDecode(data.body)["tag_name"];
},
```
